.github/workflows/packages.yml

```diff
@@ -188,7 +188,7 @@ jobs:
         done
 
         if [ ! -z "$packages" ]; then
-          ./scripts/run-docker.sh ./build-package.sh -I -a ${{ matrix.target_arch }} $packages
+          ./scripts/run-docker.sh ./build-package.sh -I -a ${{ matrix.target_arch }} --multiz $packages
```

Comment:
This affects every package. Such a change should not be made before enough tests. Please be more conservative.

I don't find any profiling report yet. Is this useful for us in the first place?

Reply:
Now checking on `libllvm`

Other packages with sources distributed as `xz` may also be tested

Reply:
I am encountering difficulties with rebuilding the docker image or `zstd` and then building a package in the same pipe.
Is there a way to do so on Github Actions? Thanks

Reply:
Only way would probably be to fork the docker image and install the extra packages in it (easier to test locally, but I understand that might not be an option).

I pushed an update to add those additional packages to the docker image, so should be possible to test once build for https://github.com/termux/termux-packages/commit/6e61626b6c319a26dd92e46b844645c70f585042 finishes